web/src/app/chat/ChatPage.tsx

```diff
@@ -116,7 +121,10 @@ export function ChatPage({
   const chatSessionIdRef = useRef<number | null>(existingChatSessionId);
 
   // LLM
-  const llmOverrideManager = useLlmOverride(selectedChatSession);
+  const llmOverrideManager = useLlmOverride(
+    user?.preferences.default_model!,
```

Comment:
if user is always non-null, can we either adjust it in the `useChatContext` typing or handle it here (e.g. by setting it to null)?